crates/protobuf_extractor/src/extract_protobuf_imports.rs

```diff
@@ -0,0 +1,62 @@
+use anyhow::{bail, Result};
+use tree_sitter::Parser;
+
+#[derive(Debug, PartialEq)]
+pub struct ProtobufSource {
+    pub imports: Vec<String>
+}
+
+impl ProtobufSource {
+    pub fn parse(source: &str, _source_path: &str) -> Result<ProtobufSource> {
+        let mut buf = Vec::default();
+        let mut parser = Parser::new();
+        parser.set_language(tree_sitter_proto::language())?;
+        let tree = match parser.parse(source, None) {
+            Some(tree) => tree,
+            None       => bail!("parse failed"),
+        };
+        let root_node = tree.root_node();
+        let mut cursor = root_node.walk();
+        let bytes = source.as_bytes();
+        for child_node in root_node.children(&mut cursor) {
+            match child_node.kind() {
+                "import" =>
+                    {
+                        for path in child_node.children_by_field_name("path", &mut child_node.walk()) {
+                            let mut quoted = path.utf8_text(bytes).unwrap().chars();
```

Comment:
this unwrap is a panic if it fails right? Can we instead add it into the `Result` type we are returning? (maybe just matching and `bail!` as above?)

Reply:
Addressed in https://github.com/bazeltools/bzl-gen-build/commit/c97d4b7c8d900f53d63d9e54182d7ffbc0fc7945 by changing to `?`